index.tsx

```diff
@@ -86,7 +86,7 @@ export const html = (text: string, opts = {}) => {
   unimplemented('html export');
 };
 
-export const mdast = (text: string, opts = {}) => {
+export const mdast: any = (text: string, opts = {}) => {
```

Comment:
ts can't infer this type because it would require exporting types from `remark`. This was my lazy way of fixing it, but we should improve this.